WalletWasabi.Gui/Helpers/NotificationHelpers.cs

```diff
@@ -15,39 +17,40 @@ public static INotificationManager GetNotificationManager()
 			return Locator.Current.GetService<INotificationManager>() ?? NullNotificationManager;
 		}
 
-		public static void Notify(string message, string title, NotificationType type, Action onClick = null)
+		public static void Notify(string message, string title, NotificationType type, Action onClick = null, object sender = null)
 		{
+			if (sender is Wallet wallet)
+			{
+				title = $"{title} - {wallet.WalletName}";
```

Comment:
I have no test it but I think it could be wise to limit the length of this string to some healthy value.

```c#
var fulltitle = $"{title} - {wallet.WalletName}";
title = fulltitle.Substring(0, Math.Min(fulltitle.Length, 50));
```

Reply:
ACK